src/components/NavBar/EditorDesignName.tsx

```diff
@@ -8,49 +8,22 @@ import {
   EditableInput,
 } from "@chakra-ui/react";
 import { BiPencil } from "react-icons/bi";
-import { useDispatch } from "react-redux";
 import { useStoreSelector } from "@/store/hooks";
-import { changeDesignName } from "@/store/reducer/courtSpecDataSlice";
 import { useEffect, useState } from "react";
-import checkName from "@/utils/checkName";
 import NameChangeFeedback from "./NameChangeFeedback";
+import { useNameCheckFeedback } from "@/hooks/useNameCheckFeedback";
```

Comment:
Do you think the name of these 2 component/custom hook are a little too similar? The first one is a modal maybe name it something modal?

Reply:
Renamed NameChangeFeedback to NameChangeAlertModal.